Jenkinsfile

```diff
@@ -141,6 +141,17 @@ node('ca-jenkins-agent') {
         junitOutput: INTEGRATION_JUNIT_OUTPUT
     )
 
+    //Upload Reports to Code Coverage
+    pipeline.createStage(
+        name: "Codecov",
+        stage: {
+            withCredentials([usernamePassword(credentialsId: 'CODECOV_ZOWE_CLI', usernameVariable: 'CODECOV_USERNAME', passwordVariable: 'CODECOV_CLI_TOKEN')]) {
+                sh "echo ${CODECOV_USERNAME} ${CODECOV_CLI_TOKEN}"
```

Comment:
Please do not try to output secret keys to the job log.
Even though Jenkins does it's best to keep them hidden, Once you figure out how to "bypass" their masking, it will be kept in the logs unless you manually delete it.

Reply:
thank you for reviewing :) I am debugging it that's why there's an echo. But I will certainly delete it.